stand/libsa/libsa.3

```diff
@@ -597,6 +597,7 @@ string after the device specification is written.
 .It Xo
 .Ft int
 .Fn devinit void
+.Xc
```

Comment:
This causes the other difference in the html output: the "Calls all the..." sentence is no longer part of the `<dt>` (which it shouldn't be).

Adding the `.Xc` brings `int devinit(void)` in line with all the other functions in this file.